source/python/astronomy/astronomy.py

```diff
@@ -5916,7 +5916,7 @@ def SearchMoonPhase(targetLon, startTime, limitDays):
     if limitDays < 0.0:
         # Search backward in time.
         if ya < 0.0:
-            ya += 360.0     # force searching forward in time, not backward
+            ya += 360.0     # force searching backward in time, not forward
```

Comment:
Thanks for spotting this! I thought I had completely deleted that comment. I need to fix this in the "template" source code (generate/template/astronomy.py), not the generated source code (source/python/astronomy/astronomy.py), for the build process to work correctly. I will do that.